### PR TITLE
feat(gitpod-cli): report errors to ide-metrics-api

### LINKED ISSUE #13707: Integrate `gp cli` with `ide-metrics-api` for at least one command
## Description
If we integrate `gp cli` with `ide-metrics-api` we can collect errors happening in the go client.

### PR DESCRIPTION
**Pull request by andreafalzetti**
## Description
Introduce error reporting from `gp cli` (startin from `gp ports list`) to `ide-metrics-api`. Errors will be available in GCP.

## Related Issue(s)

Fixes #13707

## How to test
1. Start a workspace in prev env
2. Start a workspace in prod for this PR
3. Simulate an error in prod (e.g. take down server), expose a port, and run `gp ports list`
4. Observe the logs for `ide-metrics-api` using kubectl

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by roboquat** [comment]
@andreafalzetti: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: install/installer/pkg/components/ide-metrics/configmap.go [hunk 1]
```diff
@@ -125,6 +125,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 			"vscode-workbench",
 			"vscode-server",
 			"vscode-web",
+			"gitod-cli",
```
Comment (mustard-mh) [inline install/installer/pkg/components/ide-metrics/configmap.go:128]: `gitpod-cli`

**Comment by andreafalzetti** [comment]
Closed in favour of https://github.com/gitpod-io/gitpod/pull/13752 because prev env for this PR is broken

**Pull request closed without merging**